packages/workspace/src/tasks-runner/life-cycle.ts

```diff
@@ -0,0 +1,111 @@
+import type { Task } from '@nrwl/devkit';
+import { TaskStatus } from './tasks-runner';
+import { TaskCacheStatus } from '../utilities/output';
+
+export interface LifeCycle {
+  startCommand?(): void;
+
+  endCommand?(): void;
+
+  scheduleTask?(task: Task): void;
+
+  /**
+   * @deprecated use startTasks
```

Comment:
Same minor suggestion about specific reference to `v14` if we already know that will be the version it will be removed in